src/main/webapp/WEB-INF/view/register.jsp

```diff
@@ -1,39 +1,43 @@
 <!DOCTYPE html>
 <html>
 <head>
- <title>Register</title>
- <link rel="stylesheet" href="/css/main.css">
- <style>
-   label {
-     display: inline-block;
-     width: 100px;
-   }
- </style>
+    <title>Register</title>
+    <link rel="stylesheet" href="/css/main.css">
+    <style>
+        label {
+            display: inline-block;
+            width: 100px;
+        }
+    </style>
 </head>
 <body>
 
- <nav>
```

Comment:
This is an unrelated change. The only things that is changed here is formatting. However, because of it, this PR now has a conflict.

Sticking to a consistent coding style helps to avoid problems like these, but you still need to verify your changes (using "git diff", for example) to make sure that no unrelated changes or debug code sneaks in.